model_api/cpp/utils/src/image_utils.cpp

```diff
@@ -50,6 +57,266 @@ cv::Mat resizeImageExt(const cv::Mat& mat, int width, int height, RESIZE_MODE re
         }
         break;
     }
+    case NO_RESIZE:
```

Comment:
Do we need this mode?

Reply:
I think it could be useful to state explicitly that we don't embed any interpolation into the model. E.g. if the model has been reshaped to the target resolution and no extra resize is required.